.travis.yml

```diff
@@ -7,7 +7,7 @@ services:
 language: go
 
 go:
-  - 1.8.x
+  - 1.9.x
```

Comment:
I think Travis is having issues with `1.9.x` right now, but I was able to get `1.9` to work on another project.

```
$ GIMME_OUTPUT=$(gimme 1.9.x | tee -a $HOME/.bashrc) && eval "$GIMME_OUTPUT"
I don't have any idea what to do with '1.9.x'.
  (using type 'auto')
```

Reply:
Once https://github.com/travis-ci/travis-build/pull/1148 is merged, hopefully it should just work :)